extensions/wikia/CommunityPage/templates/pageHeader.mustache

```diff
@@ -1,8 +1,6 @@
 <div class="community-page-header {{#heroImageUrl}}community-page-header-cover" style="background-image: url({{heroImageUrl}});{{/heroImageUrl}}">
 	<div class="community-page-header-content">
 		<h1 class="community-page-heading">{{{headerWelcomeMsg}}}</h1>
+		<h1 class="community-page-subheading">{{{subheaderWelcomeMsg}}}</h1>
```

Comment:
pages shouldn't have more than one `h1` - please use `h2` or `h3` as it is a subheader, not a main one